### ISSUE TITLE
calculate time on site

### ISSUE THREAD

**Issue by mikestaub** [seanmiller802/BrowserTime#11]
Do the percentages treat each website view the same?

Meaning if I spend 2 hours on twitter but 10 seconds on cnn does it say 50% social and 50% news?

**Comment by seanmiller802** [comment]
Yes it does currently. I want to add this but I haven't done it yet bc it would increase the scope of the project by quite a bit I think since I would need to start a timer on each page visit and have some algo for figuring out if someone is actively interacting with the page or not. Would love suggestions on how to approach this though.

**Comment by mikestaub** [comment]
I would use the `visibilitychange` event to log when the user starts and ends their session on a page and just log these events. You can also use `focus` and `blur` on the window to do this I believe.
